src/main/java/com/CezaryZal/api/limits/manager/checker/LimitsChecker.java

```diff
@@ -0,0 +1,18 @@
+package com.CezaryZal.api.limits.manager.checker;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class LimitsChecker {
+
+    private final int PORTION_OF_DRINK = 250;
+    private final double LIMIT_OF_EAT_KCAL = 0.05; // 5%
+
+    public boolean checkIsAchievedDrink(int drinkDemandPerDay, int portionsDrink) {
+        return drinkDemandPerDay <= portionsDrink * PORTION_OF_DRINK;
```

Comment:
drinkDemandPerDay <= portionsDrink * PORTION_OF_DRINK; ja bym to wrzuciłdo metody, którje opis mówiłby co tu sie dzieje

Reply:
Ok, wyjasnione i poprawione